src/classes/ElevateCaptureGroup.cls

```diff
@@ -0,0 +1,4 @@
+public interface ElevateCaptureGroup {
+    UTIL_Http.Response add(ElevateTokenizedGift tokenizedGift, String groupId);
+    UTIL_Http.Response create();
```

Comment:
I think add should return an AuthorizedGift, and create should return a CaptureGroupId.

Reply:
@kenlewis-sfdo Agreed. That's coming next. Just wanted to get the initial code stood up so I could write the tests. But I will be updating that piece today and update the tests!